Source/Core/Core/DSP/DSPCore.h

```diff
@@ -308,6 +308,9 @@ struct SDSP
 
   // This one doesn't really belong here.
   u8* cpu_ram;
+
+  // This is used by DSPJitRegCache to store EBP.
+  u64 ebp_store;
```

Comment:
That may or may not have been to aid debugging on x86. I doubt it is needed or useful on amd64. In fact, my JitIR does not restore ebp for function calls.